### PR TITLE
added distant chat settings for store/load

### PR DESCRIPTION
**Pull request by defnax**
added distant chat settings, it works with the changes from pr #1680
i tested storing works now, not sure if all the changes from me is correct .

### CODE REVIEW

File: libretroshare/src/pqi/p3historymgr.cc [hunk 1]
```diff
@@ -99,11 +102,25 @@ void p3HistoryMgr::addMessage(const ChatMessage& cm)
 			enabled = true;
 		}
 
-		if(cm.chat_id.isDistantChatId())
+		if(cm.chat_id.isDistantChatId()&& mDistantEnable == true)
```
Comment (csoler) [inline libretroshare/src/pqi/p3historymgr.cc:105]: This can be 

if(cm.chat_id.isDistantChatId() && mDistantEnable)

... since it is a boolean already.

Comment (csoler) [reply libretroshare/src/pqi/p3historymgr.cc:105]: did I write this myself??

**Comment by csoler** [comment]
Can you plz remove .gitignore from the pull request? thx

File: libretroshare/src/pqi/p3historymgr.cc (hunk 1 above)
Comment (defnax) [reply libretroshare/src/pqi/p3historymgr.cc:105]: No thats added by me, i saw from other example by default enable
` if (cm.chat_id.isPeerId() && mPrivateEnable == true) `

**Pull request merged**